lib/connectors/gitlab/connector.rb

```diff
@@ -48,8 +44,15 @@ def configurable_fields
         }
       end
 
-      def sync_content(_params = {})
-        extract_projects
+      def sync_content(connector)
+        sink = Utility::Sink::CombinedSink.new(
+          [
+            Utility::Sink::ConsoleSink.new,
```

Comment:
I'm always curious what is the purpose of `ConsoleSink`? Just log something in the console?